buildchain/buildchain/salt_tree.py

```diff
@@ -651,7 +651,8 @@ def _get_parts(self) -> Iterator[str]:
     Path('salt/metalk8s/volumes/init.sls'),
     Path('salt/metalk8s/volumes/prepared/init.sls'),
     Path('salt/metalk8s/volumes/prepared/installed.sls'),
-    Path('salt/metalk8s/volumes/provisioned/init.sls'),
+    Path('salt/metalk8s/volumes/prepared/files/metalk8s-sparse-volume@.service'),
```

Comment:
About commit message: really good one :+1: 

On

> to include the fix from higher versions of `losetup`

Maybe add [a link to the fix](https://github.com/karelzak/util-linux/commit/f7e21185087c30c5f13bf92009238e316f8c1cf4) and/or mention in which version it was fixed (v2.25 IIRC), maybe when we migrate to CentOS 8 we can get rid of this workaround.

Reply:
Done!